### PR TITLE
Use `Post#public_id` instead of `Post#id` for public page

### PR DESCRIPTION
**Pull request by tricknotes**

### CODE REVIEW

**Comment by mtsmfm** [comment]
Awesome :shipit: :star2:

File: scripts/convert_html2md.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 parser.banner = <<BANNER
 Usage
-  $ bin/rails r scripts/convert_html2md.rb [options] <fqdn> [id]
+  $ bin/rails r scripts/convert_html2md.rb [options] <fqdn> [public_id]
```
Comment (okkez) [inline scripts/convert_html2md.rb]: `$ bin/rails r scripts/convert_html2md.rb [runner options] -- [options] <fqdn> [public_id]`

Could you fix like above?
I forgot to fix it.

Comment (tricknotes) [inline scripts/convert_html2md.rb]: :wrench: by https://github.com/bm-sms/daimon-news-multi-tenant/commit/364e6cc53e816738ebf8a110c5915b6a383e4961

**Comment by okkez** [comment]
Awesome!

**Pull request merged**